cfgov/templates/welcome.html

```diff
@@ -3,14 +3,42 @@
 
 {% block furniture %}
     <div class="content-wrapper">
+
         <h1>{% trans 'Welcome' %} <br/>
-        You've logged in to the consumerfinance.gov editorial tools.	
+        {% if 'build' in request.site.hostname %}
```

Comment:
1. As noted below, I don't see a point in restricting the helpful links below to any particular server.
2. If we're taking servers other than Build and Content into consideration, then we probably need to reevaluate this whole thing. "Demo Wagtail" is meant to specifically refer to the playground on Build.